packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -467,6 +472,35 @@ class RefreshIndicatorState extends State<RefreshIndicator> with TickerProviderS
       });
   }
 
+  // Method to verify the isRefreshing attribute and show/hide indicator
+  bool _checkIsRefreshing(bool? isRefreshing) {
+    if (isRefreshing != null) {
+      if (isRefreshing) {
+        _showProgressIndicator();
+      } else {
+        //Hides indicator only if running
+        if (mounted && _mode == _RefreshIndicatorMode.refresh) {
+          _dismiss(_RefreshIndicatorMode.done);
+        }
+      }
+      return isRefreshing;
+    } else {
+      return false;
+    }
```

Comment:
This `else` is not required, you can return it directly:

```suggestion
  } 
  return false;
```

Reply:
Thank you!